tarea/task-2-class6-salary-form-bootstrap/task-2-class-10-salary-index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../../node_modules/bootstrap/dist/css/bootstrap.min.css">
```

Comment:
You should change this to bootstrap CDN. Remember you ignored ``node_modules`` with ``.gitignore```so if you deploy your app to gh-pages it will try to find ``node_modules`` folder but it doesn't exist. Also, gh-pages doesn't work with node_modules.

Tell me if you understood this.